### PR TITLE
Logs toolbar style adjustment to mock

### PR DESCRIPTION
**Pull request by zreigz**
toolbar

### CODE REVIEW

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 $logs-toolbar-color-black: #000;
 $logs-toolbar-color-white: #fff;
+$logs-toolbar-text-font-size: 17px;
+$logs-toolbar-title-font-weight: 500;
+$logs-toolbar-select-margin-bottom: 15px;
+$logs-toolbar-select-margin-left: auto;
```
Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: This (and line 19) does not need to be in a variable. It is used only once and is pretty standard way of margining.

Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: For 15px you could use 2*$baseline-grid. Will it look okay? The reason is that we should not use magic numbers unless really needed, and base everything on baseline grid.

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 $logs-toolbar-color-black: #000;
 $logs-toolbar-color-white: #fff;
+$logs-toolbar-text-font-size: 17px;
```
Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: Can you use any of font sizes from https://github.com/angular/material/blob/HEAD/src/core/style/variables.scss and define the font in src/app/frontend/_variables.scss ?

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss [hunk 3]
```diff
@@ -14,6 +14,10 @@
...
+$logs-toolbar-title-font-weight: 500;
```
Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: I checked demo and without this 500 it looks better. How about removing this?

**Comment by bryk** [comment]
Looks really nice. PTAL

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss (hunk 3 above)
Comment (zreigz) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: I removed this.

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss (hunk 2 above)
Comment (zreigz) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: done

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss (hunk 1 above)
Comment (zreigz) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: done

**Comment by zreigz** [comment]
PTAL

File: src/app/frontend/_variables.scss [hunk 4]
```diff
@@ -12,6 +12,20 @@
...
+// Function
+//-- Must be defined before variables
+@function rem($multiplier) {
+  $font-size: 10px;
+  @return $multiplier * $font-size;
+}
+
+// Typography
+// ------------------------------
+$font-family: Roboto, 'Helvetica Neue', sans-serif !default;
```
Comment (bryk) [inline src/app/frontend/_variables.scss]: You dont need $font-family. All our texts are already with this font family.

File: src/app/frontend/_variables.scss [hunk 5]
```diff
@@ -12,6 +12,20 @@
...
+$font-size:   10px;
```
Comment (bryk) [inline src/app/frontend/_variables.scss]: You dont need to have it defined here. On line 18 is enough.

File: src/app/frontend/logs/logstoolbar/logstoolbar.scss [hunk 6]
```diff
@@ -33,3 +35,13 @@ md-toolbar {
     }
   }
 }
+
+.kd-logs-toolbar-text {
+  font-family: $font-family;
```
Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.scss]: Font family is already defined by angular material, so it is safe to remove it from here.

**Comment by bryk** [comment]
Awesome. Last comments and let's merge. PTAL

**Comment by zreigz** [comment]
PTAL

**Comment by bryk** [comment]
LGTM

**Pull request merged**